### DISCUSSION TITLE
Calculate token usage in Python SDK

### DISCUSSION THREAD

**Discussion by nehalakhani** [microsoft/semantic-kernel#8702] [Q&A] [answered]
Hello Team,

Is there direct method available in Semantic Kernel to calculate the tokens - Prompt Tokens and Completions tokens. Please suggest various ways to calculate token in Python SDK

**Comment by evchaki** [comment, marked as answer]
@nehalakhani  - this is in the metadata in the function result.  You can store the metadata and do something with that (prompt tokens and comp tokens).
